### PR TITLE
Add contributor/reviewer checklist

### PR DESCRIPTION
**Pull request by OYTIS**

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -11,3 +11,15 @@ Previously, some older branches were also regularly supported, and while they sh
 * morty
 
 If you are developing with meta-updater, it may be helpful to read the README and other documentation for link:README.adoc[this repo], https://github.com/advancedtelematic/aktualizr[aktualizr], and the https://github.com/advancedtelematic/updater-repo/[updater-repo], particularly the sections about development and debugging.
+
+== Contributer checklist
```
Comment (lbonn) [inline CONTRIBUTING.adoc]: "contributor"

Comment (OYTIS) [reply CONTRIBUTING.adoc]: Thanks.

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -11,3 +11,15 @@ Previously, some older branches were also regularly supported, and while they sh
 * morty
 
 If you are developing with meta-updater, it may be helpful to read the README and other documentation for link:README.adoc[this repo], https://github.com/advancedtelematic/aktualizr[aktualizr], and the https://github.com/advancedtelematic/updater-repo/[updater-repo], particularly the sections about development and debugging.
+
+== Contributor checklist
+
+* OTA-enabled build succeeds at least for one platform, the resulted image boots. This check is absolutely necessary for every pull request unless it only touches documentation.
+* If your change is touching platform code (like `classes/sota_<platform>.bbclass`), please check build and update on this particular platform.
+* oe-selftest succeeds. To test meta-updater, run `oe-selftest -r updater` from a build directory with `MACHINE` set to `qemux86-64`.
+* Updates are forwards- and backwards-compatible. You should be able to update an OTA-enabled build before the change is applied to the version with change applied and vice versa. One should pay double attention to the compatibility when bootloader code is affected.
```
Comment (lbonn) [inline CONTRIBUTING.adoc]: In the current state of aktualizr, rolling back will probably won't work, right? It can still be a worthwhile goal.

Comment (OYTIS) [reply CONTRIBUTING.adoc]: Rolling back might not work when if we update the storage schema, yes. But
a) I hope we'll fix that soon.
b) External contributors won't update aktualizr probably (this documents applies to internal contributors as well of course).

**Comment by patrickvacek** [comment]
+1, thanks for writing this up.

**Pull request merged**